src/makeCalculator.js

```diff
@@ -4,7 +4,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    add(number) {
+      result += number;
+    },
+    subtract(number) {
+      result -= number;
+    },
+    multiply(number) {
+      result *= number;
+    },
+    divide(number) {
+      result /= number;
+    },
+
+    reset() {
+      result = 0;
+
+      return this;
+    },
+
+    operate(callback, number) {
```

Comment:
Use the method shorthand syntax for defining object methods. Replace `operate: function(callback, number) {` with `operate(callback, number) {`.